integrations/database.go

```diff
@@ -0,0 +1,37 @@
+package integrations
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	_ "github.com/lib/pq"
+)
+
+var host = os.Getenv("DB_HOST")
+var port = os.Getenv("DB_PORT")
+var user = os.Getenv("DB_USER")
+var password = os.Getenv("DB_PASSWORD")
+var dbname = os.Getenv("DB_NAME")
+
+var DB *sql.DB
```

Comment:
Of course it's a very good practise :) and tbh I wouldn't leave this as is if I had the familiarity with Go and a bit more time.
Using Globals which are shared between different packages/files make the code difficult to maintain and it's possible to lead to several defects.
Each time you have to make a change related to this variable you have to check every file using it making sure that won't affect/break it.
I would say that proper way of handling this would be changing the scope of the variable to local for this file and using a getter function in order to make it accessible outside of the package.